setup.py

```diff
@@ -1,33 +1,6 @@
 #!/usr/bin/env python
 from setuptools import setup  # isort:skip
-import os
-from itertools import chain
 
 from extension_helpers import get_extensions
 
-try:
-    # Recommended for setuptools 61.0.0+
-    # (though may disappear in the future)
-    from setuptools.config.setupcfg import read_configuration
-except ImportError:
-    from setuptools.config import read_configuration
-
-################################################################################
-# Programmatically generate some extras combos.
-################################################################################
-extras = read_configuration("setup.cfg")['options']['extras_require']
-
-# Dev is everything
-extras['dev'] = list(chain(*extras.values()))
-
-# All is everything but tests and docs
-exclude_keys = ("tests", "docs", "dev")
-ex_extras = dict(filter(lambda i: i[0] not in exclude_keys, extras.items()))
-# Concatenate all the values together for 'all'
-extras['all'] = list(chain.from_iterable(ex_extras.values()))
-
-setup(
-    extras_require=extras,
-    use_scm_version={'write_to': os.path.join('sunpy', '_version.py')},
-    ext_modules=get_extensions(),
-)
+setup(ext_modules=get_extensions())
```

Comment:
We don't even need this, you can do:
```
[extension-helpers]
use_extension_helpers = true
```

https://extension-helpers.readthedocs.io/en/latest/using.html